.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+### Description
+<!-- A description of the bug or feature -->
+
+### Steps to Reproduce
+<!-- List of steps, sample code, failing test or link to a project that reproduces the behavior.
+     Make sure you place a stack trace inside a code (```) block to avoid linking unrelated issues -->
+
+### System Configuration
+<!-- Tell us about the environment where you are experiencing the bug -->
+
+- ImageMagick version:
+- RMagick version:
```

Comment:
@Watson1978 should this have Ruby version, too?

Reply:
Sounds good :)